share/spack/gitlab/cloud_pipelines/stacks/e4s-power/spack.yaml

```diff
@@ -0,0 +1,426 @@
+  - tasmanian
+  - tau +mpi +python
+  - trilinos@13.0.1 +belos +ifpack2 +stokhos
+  - turbine
+  - umap
+  - umpire
+  - upcxx
+  - wannier90
+
+  # CUDA
+  - amrex +cuda
+  - arborx +cuda ^kokkos +wrapper
+  - cabana +cuda ^kokkos +wrapper +cuda_lambda +cuda
+  - caliper +cuda
+  - chai ~benchmarks ~tests +cuda ^umpire ~shared
+  - ecp-data-vis-sdk +cuda cuda_arch=70 +adios2 +hdf5 ~paraview +vtkm +zfp # +paraview fails:  FAILED: VTK/Filters/Statistics/CMakeFiles/FiltersStatistics-objects.dir/vtkPCAStatistics.cxx.o; /tmp/ccjmJhb6.s: Assembler messages: /tmp/ccjmJhb6.s:260012: Error: invalid machine `power10'
+  - flecsi +cuda
+  - flux-core +cuda
+  - ginkgo +cuda
+  - heffte +cuda
+  - hpctoolkit +cuda
+  - hpx max_cpu_count=512 +cuda
+  - hypre +cuda
+  - kokkos +wrapper +cuda
+  - kokkos-kernels +cuda ^kokkos +wrapper +cuda
+  - magma +cuda
+  - mfem +cuda
+  - omega-h +cuda
+  - papi +cuda
+  - petsc +cuda
+  - py-torch +cuda
+  - raja +cuda
+  - slate +cuda
+  - slepc +cuda
+  - strumpack ~slate +cuda
+  - sundials +cuda
+  - superlu-dist +cuda
+  - tasmanian +cuda
+  - tau +mpi +cuda
+  - trilinos@13.4.0 +belos +ifpack2 +stokhos +cuda
+  - umpire ~shared +cuda
+  - parsec +cuda
+
+  # CPU FAILURES
+  # - archer                  # llvm@8
+  # - bricks                  # bricks
+  # - geopm                   # geopm
+  # - loki                    # loki
+  # - precice                 # precice
+  # - pruners-ninja           # pruners-ninja
+  # - variorum                # Intel/variorum_cpuid.c:11:5: error: impossible constraint in 'asm'
+  # --
+  # bricks: VSBrick-7pt.py-Scalar-8x8x8-1:30:3: error: 'vfloat512' was not declared in this scope
+  # fltk: /usr/bin/ld: ../lib/libfltk_png.a(pngrutil.o): in function `png_read_filter_row': pngrutil.c:(.text.png_read_filter_row+0x90): undefined reference to `png_init_filter_functions_vsx'
+  # geopm: libtool.m4: error: problem compiling CXX test program
+  # llvm@8: clang/lib/Lex/Lexer.cpp:2547:34: error: ISO C++ forbids declaration of 'type name' with no type [-fpermissive]
+  # loki: include/loki/SmallObj.h:462:57: error: ISO C++17 does not allow dynamic exception specifications
+  # precice: /tmp/ccYNMwgE.s: Assembler messages: /tmp/ccYNMwgE.s:278115: Error: invalid machine `power10'
+  # pruners-ninja: test/ninja_test_util.c:34: multiple definition of `a';
+
+  # CUDA FAILURES
+  # - bricks +cuda              # bricks
+  # - dealii +cuda              # fltk
+  # --
+  # bricks: VSBrick-7pt.py-Scalar-8x8x8-1:30:3: error: 'vfloat512' was not declared in this scope
+
+
+  mirrors: { "mirror": "s3://spack-binaries/develop/e4s-power" }
+
+  gitlab-ci:
+
+    script:
+      - uname -a || true
+      - grep -E 'vendor|model name' /proc/cpuinfo 2>/dev/null | sort -u || head -n10 /proc/cpuinfo 2>/dev/null || true
+      - nproc
+      - curl -Lfs 'https://github.com/JuliaBinaryWrappers/GNUMake_jll.jl/releases/download/GNUMake-v4.3.0+1/GNUMake.v4.3.0.x86_64-linux-gnu.tar.gz' -o gmake.tar.gz
```

Comment:
I think maybe this is for x86_64 only. Either `ci rebuild` needs an option for the make stuff or this may need to get make in another way.

Reply:
yes, you are right. will fix.